script/build-libgit2-static.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-ROOT="$(cd "$0/../.." && echo "${PWD}")"
+ROOT="$(cd -- "$(dirname -- "$0")" && git rev-parse --show-toplevel)"
```

Comment:
I like the `echo` more than the `git rev-parse...` in case the code is ever without the git information, say from a git archive, e.g.

Reply:
fair enough. as long as there is _a_ solution. i approved the other PR, let me see if we can get that one merged.